### ISSUE TITLE
Copy to clipboard doesn't work as expected on mobile

### ISSUE THREAD

**Issue by ajpeddakotla** [hypothesis/product-backlog#858]
###Problem 
I wasn't able to reproduce this on my Android in Chrome, but others have reported that the copy to clipboard sharing functionality does not work. I do see however that the share link that is given to users from the top header doesn't show the clipboard icon at all. 

![screenshot_20181030-170304](https://user-images.githubusercontent.com/7463523/47753519-d188a100-dc65-11e8-803d-d784676edf16.png)

**Comment by dwhly** [comment]
I was able to reproduce this in iOS Chrome.

**Comment by klemay** [comment]
Reproduced on Chrome for iOS. A couple of notes:
 - I had to tap on the "Share" button a few times before the share link appeared
- I had to tap on the Clipboard icon several times before anything happened
- When I tap on the Clipboard icon, rather than automatically copying the link to my clipboard, I am prompted to "Select and copy to share"
- When I try to select the link text and copy, I am unable to 

I took a screencast, which I uploaded here: https://github.com/hypothesis/support/blob/master/videos/ScreenRecording_10-31-2018%2013-51-16.MP4

**Comment by xolotl** [comment]
Confirming that the copy to clipboard capability is missing from the page share UX at @ajpeddakotla mentions in this issue, and on iOS browsers like Chrome and FF, I'm not able to even select all of the link there so sharing the page annotation link seems blocked on mobile.

**Comment by robertknight** [comment]
> Confirming that the copy to clipboard capability is missing from the page share UX at @ajpeddakotla mentions in this issue

Can we treat that as a separate issue please? It has always been the case that the page share dialog was a plain text field with no "Copy" button.

When I saw this issue I was reminded that we have a [mobile Safari-specific workaround](https://github.com/hypothesis/h/blob/0c81106a6cd9f9380f6c8b3258eb64ef391d00b1/h/static/scripts/controllers/copy-button-controller.js#L20) in our "Copy to clipboard" button in h which may be relevant. Whoever looks into this should check the Git history and linked issues for that file in case they are useful.

**Comment by klemay** [comment]
I am now able to copy to clipboard from iOS, but the behavior is a bit unexpected. 

I am trying this using https://via.hypothes.is/https://web.hypothes.is/.

In Safari for iOS, tapping on the share icon for an annotation works as expected:
![IMG_6F0BC79BF1F0-1](https://user-images.githubusercontent.com/12087705/98568253-e352a500-227e-11eb-87b1-50de693505cf.jpeg)

...but when you hit the "copy" target, it highlights the URL, extending past the button, and brings up the native iOS menu [note that you can select "copy" here to copy the URL:

![IMG_1912](https://user-images.githubusercontent.com/12087705/98568461-1eed6f00-227f-11eb-8342-217c13b257e1.PNG)

you can also tap anywhere on the screen to dismiss the iOS-native menu, and tap the copy icon to successfully copy the link:

![IMG_1913](https://user-images.githubusercontent.com/12087705/98568639-54925800-227f-11eb-9784-480c905c9c9b.PNG)

Same behavior happens in Chrome for iOS.
